[8.8] Vulnerability management integration docs (#3162)
elastic/security-docs: docs/cloud-native-security/vuln-management-faq.asciidoc

*Does CNVM have an impact on the user's cloud expenses?*
Yes, CNVM creates additional cloud expenses, since scanning involves provisioning a new virtual machine to conduct the scan.

*Does CNVM also scan the new AWS EC2 instances that it creates?*
Yes, CNVM scans all AWS EC2 instances in every scan cycle, including any created by the integration.

*Does CNVM scan AWS EC2 instances with encrypted volumes?*
CNVM does not currently support scanning of encrypted volumes. This limitation will be addressed in future releases.

*Does CNVM prevent multiple installations in a single region?*
No, CNVM does not currently prevent redundant deployment to the same region.

*What volume types and file systems does CNVM support?*
CNVM supports all AWS EBS volume types and works with `ext4` and `xfs` file systems.

*Does CNVM scan stopped EC2 instances?*
Yes, CNVM scans all EC2 instances, whether they are running or stopped, to ensure comprehensive vulnerability detection.

*What AWS permissions does the user require to run the CloudFormation template for CNVM onboarding?*

> To run the CloudFormation template for CNVM onboarding, you need an AWS user account with permissions to perform the following actions: run CloudFormation templates, create IAM Roles and InstanceProfiles, and create EC2 SecurityGroups and Instances.

Reviewer: Customers probably need to know this _before_ they try to start running the template. Include this info as prerequisites or requirements on the "Get started with CNVM" page?

> A: To run the CloudFormation template for CNVM onboarding, the user needs to have an AWS user account with permissions to perform the following actions: run CloudFormation templates, create IAM Roles and InstanceProfiles, and create EC2 SecurityGroups and Instances.

Author: ```suggestion
To run the CloudFormation template for CNVM onboarding, you need an AWS user account with permissions to perform the following actions: run CloudFormation templates, create IAM Roles and InstanceProfiles, and create EC2 SecurityGroups and Instances.
```
